testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-adls/core-site-abfs-template.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<configuration>
+    <property>
+        <name>fs.defaultFS</name>
+        <value>hdfs://hadoop-master:9000</value>
+    </property>
+
+    <property>
+      <name>fs.azure.account.key.%ABFS_ACCOUNT%.dfs.core.windows.net</name>
+      <value>%ABFS_ACCESS_KEY%</value>
+    </property>
+
+    <!-- OOZIE proxy user setting -->
+    <property>
+        <name>hadoop.proxyuser.oozie.hosts</name>
+        <value>*</value>
+    </property>
+    <property>
+        <name>hadoop.proxyuser.oozie.groups</name>
+        <value>*</value>
```

Comment:
all these seem redundant. they are a copy so that we can apply `fs.azure.account.key.%ABFS_ACCOUNT%.dfs.core.windows.net`, which seems the only config property that actually matters

we should use `apply-config-overrides.sh` instead like
```
apply-site-xml-override /etc/hive/conf/hive-site.xml "/docker/presto-product-tests/conf/environment/singlenode-hdp3/hive-site-overrides.xml"
```